src/utils/getResponse.ts

```diff
@@ -56,8 +56,15 @@ export const getResponse = async <
       handler: null,
       response: null,
     }
+  } else if (relevantHandlers.length > 1) {
+    // TODO: this is probably not the best approach... we need some way to limit the result set of relevantHandlers to 1 of each type.
+    // Is this really even possible though? Will one request ever _really_ match multiple handlers?
```

Comment:
If a request has a matching handler and that handler returns a mocked response, _only that handler_ will be used. Even if there are other handlers that match, they are resolved in the order they were attached. A single request cannot respect multiple matching handlers, this is a no-op. 

Why do we need to have this logic, though?